web/app/util/objects.ts

```diff
@@ -4,3 +4,25 @@ export const withoutNullValues = <T extends object>(obj: T): Partial<T> =>
       ([_key, value]) => value !== null && value !== undefined
     )
   ) as Partial<T>
+
+export const isEmptyModelObject = (obj: any): Boolean => {
+  if (obj === undefined || obj === null) {
+    return true
+  }
+  if ('length' in obj && obj.length > 0) {
+    return false
+  }
+  if (typeof obj === 'object') {
+    for (const [key, value] of Object.entries(obj)) {
+      if (key === '$class') {
+        continue
+      }
+      if (!isEmptyModelObject(value)) {
+        return false
+      }
```

Comment:
Missaanko jotain vai eiks tää ois rekursion hengessä vaan näin?
```suggestion
      return isEmptyModelObject(value)
```

Reply:
Toi menis rikki esim. siinä tapauksessa että meillä on objekti joka näyttää tältä `{[], [{foo: 'jotain'}]}`
Eli jos rakenteessa on yksikin ei-tyhjä asia niin koko objekti on ei-tyhjä.